flight/Modules/Stabilization/stabilization.c

```diff
@@ -632,7 +810,7 @@ static void stabilizationTask(void* parameters)
 		uint16_t max_safe_rate = PIOS_SENSORS_GetMaxGyro() * 0.9f;
 
 		//Run the selected stabilization algorithm on each axis:
-		for(uint8_t i=0; i< MAX_AXES; i++)
+		for (uint8_t i = 0; i < MAX_AXES; i++)
 		{
```

Comment:
If you're cleaning it up, move the curly-bracket up where it belongs. :D

Reply:
:D